src/protocols/ssh/client.c

```diff
@@ -44,6 +44,9 @@ int guac_client_init(guac_client* client) {
     guac_ssh_client* ssh_client = calloc(1, sizeof(guac_ssh_client));
     client->data = ssh_client;
 
+    /* Init clipboard */
+    ssh_client->clipboard = guac_common_clipboard_alloc();
```

Comment:
I don't think we should avoid using the clipboard provided by the terminal:

* The underlying issue here likely affects all terminal-related protocols, not just SSH.
* Switching to a dedicated terminal here would mean that options like `disable-copy` and `disable-paste` would need to be manually reimplemented.

Instead, this should be done at the terminal level. If there needs to be an option for this, then a terminal option can be provided, though I suspect we can safely normalize to UNIX-style newlines for all terminal protocols because of how the terminal clipboard functions.